guides/common/assembly_configuring-and-setting-up-remote-jobs.adoc

```diff
@@ -34,8 +34,10 @@ Use one of the following methods to distribute the public SSH key from {SmartPro
 . xref:distributing-ssh-keys-for-remote-execution-manually_{context}[].
 . xref:using-the-api-to-obtain-ssh-keys-for-remote-execution_{context}[].
 . xref:configuring-a-kickstart-template-to-distribute-ssh-keys-during-provisioning_{context}[].
+ifdef::satellite[]
 . For new {Project} hosts, you can deploy SSH keys to {Project} hosts during registration using the global registration template.
 For more information, see {ManagingHostsDocURL}registering-a-host-to-project-using-the-global-registration-template_managing-hosts[Registering a Host to {ProjectName} Using the Global Registration Template].
```

Comment:
```suggestion
For more information, see {ManagingHostsDocURL}registering-a-host_managing-hosts[Registering a Host to {ProjectName} Using the Global Registration Template].
```

Reply:
I've fixed this & also adjusted the anchor (replacing ~{context}~ with `managing-hosts`).

Rebased to master.

Reply:
Thank you. 
It bothers me that the link checker did not catch this!!! 
I'll have to remember to ping @lzap about it when he returns.

Reply:
This seems to be resolved for now. Anything else I can do @melcorr ?